.github/workflows/native.yml

```diff
@@ -31,6 +33,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: sbt/sbt
+          ref: 1.8.x
```

Comment:
@eed3si9n I think it would be a good idea to define the branch here. Like now I would like to use the current stable one (1.8.x).
Would it make sense to introduce a 1.8.x and 1.9.x branch  here in the sbt/sbtn-dist repository as well, which refere to the according sbt branch in its `native.yml`? And even remove the `develop` branch?
What do you think?

Reply:
Yea, thus far I've only had to build sbtn for fresh new minor releases, so assuming the default branch has been ok.